### ISSUE TITLE
Why track the loss and accuracy on test mode?

### ISSUE THREAD

**Issue by PabloRR100** [kuangliu/pytorch-cifar#56]
What is the point of tracking the loss and accuracy iterating over the test set?

If your model is already trained, why don't you simple compute the accuracy over the entire test set and that's it?
